charter.html

```diff
@@ -171,12 +171,13 @@ <h3>Focus for the 2022 - 2024 Charter</h3>
 						<li>Rewriting existing guidelines,</li>
 						<li>Adding guidance to address known gaps in WCAG 2,</li>
 						<li>Creating new guidelines for emerging technologies,and</li>
-						<li>Complex conformance challenges, such as third-party content, website conformance, scoring, accessibility statements, etc.</li>
+						<li>Complex conformance challenges, such as third-party content, conformance, scoring, accessibility statements, etc.</li>
```

Comment:
Complex conformance challenges, including conformance? Suggest "Complex challenges" at start of sentence.